src/components/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Header from "../Header";
+import Footer from "../Footer";
+import Sidebar from "./Sidebar";
+import MainContent from "./MainContent";
+
+const Dashboard = () => {
+  return (
+    <div className="mainDashboard">
+      <Header></Header>
```

Comment:
Please render the header as <Header />, because we do not need to pass react children in header.

Reply:
done